Esri/arcgis-runtime-samples-ios
# New: Map image layer tables

## Description

This PR implements `Map image layer tables` in `Layers` category.
Branch: https://github.com/Esri/arcgis-runtime-samples-ios/tree/Viv/manageMapImage

## Linked Issue(s)

- `common-samples/issues/3262`

## How To Test

Tap on bottom toolbar and select some of the available comments. Notice the first result will be cyan.
